### PR TITLE
refactor(community): improvements to insights

### PR DESCRIPTION
**Pull request by NathanInnes**
Fixes N/A

## Summary of changes
- Community Insights section revamped.
- 2 extra font weights added.
- Tailwind config height utility and discord darker colour added.
- New card component
- Removed discord section

### Desktop
image
### iPad/Tablet
image
### iPhone 8/Mobile Medium
image
image

### CODE REVIEW

File: components/home/Community.tsx [hunk 1]
```diff
@@ -141,24 +92,51 @@ export function Community(): JSX.Element {
...
+                <div className="my-12">
+                    <Card bgColour="gray-100" borderColour="discord" className="shadow-lg">
+                        <CardTitle>
+                            <FontAwesomeIcon className="mr-2 text-discord" icon={faDiscord} size="sm" />
+                            Discord
+
+                        </CardTitle>
+                        <CardBody>
+                            Our Discord server is where we plan the entirety of our projects and provide most of our support.
+                            Join us to chat with other members of the community, get started with contributing or ask us a question!
```
Comment (ErickSharp) [inline components/home/Community.tsx:125]:
```suggestion
                            Join us to chat with other members of the community, get started with contributing, or ask us a question!
```
Serial comma

**Review by ErickSharp — APPROVED** [review]
LGTM looks amazing.

File: components/home/Community.tsx [hunk 2]
```diff
@@ -141,24 +92,51 @@ export function Community(): JSX.Element {
...
-
-                <div className="mx-auto flex flex-col lg:flex-row lg:justify-center divide-y lg:divide-x lg:divide-y-0 divide-gray-500">
+        <section className="flex flex-col lg:flex-row justify-between items-center bg-gray-50 text-blue-dark-contrast">
+            <div className="mt-10 px-page lg:px-24 flex flex-col">
+                <span className="w-24 h-2 bg-teal-light-contrast rounded-full mb-3" />
+                <h1 className="text-5xl pr-2 font-bold">
+                    Community Insights
+                </h1>
+                <p className="text-xl text-gray-800 max-w-prose py-4">
+                    Discover the extensive community behind every FlyByWire Simulations aircraft - a vibrant and active online group who prioritises collaborative work and openness.
```
Comment (ErickSharp) [inline components/home/Community.tsx:102]:
```suggestion
                    Discover the extensive community behind every FlyByWire Simulations aircraft - a vibrant and active online group that prioritises collaborative work and openness.
```
Forgot one last minor grammar fix.

**Comment by NathanInnes** [comment]
On hold until partners section is complete

**Pull request merged**